src/actions/sso/paymentRequest.ts

```diff
@@ -34,11 +36,25 @@ export const consumePaymentRequest = (
     registry as JolocomRegistry,
   )
 
+  const { did: offerorDid, publicProfile } = await registry.resolve(
+    keyIdToDid(paymentRequest.issuer),
+  )
+
+  const parsedProfile = publicProfile
+    ? omit(['id', 'did'], publicProfile.toJSON().claim)
+    : {}
+
+  const requesterInfo = mergeRight(
+    { did: offerorDid },
+    { publicProfile: parsedProfile },
+  )
```

Comment:
What is the reason for using `mergeRight` here? isn't it the same as `cont requesterInfo = { did: offerorDid , publicProfile: parsedProfile }`?